src/libstd/sys/unix/thread.rs

```diff
@@ -342,7 +342,10 @@ fn min_stack_size(attr: *const libc::pthread_attr_t) -> usize {
     static mut __pthread_get_minstack: Option<F> = None;
 
     INIT.call_once(|| {
-        let lib = DynamicLibrary::open(None).unwrap();
+        let lib = match DynamicLibrary::open(None) {
+            Ok(l) => l,
+            Err(..) => return,
+        };
```

Comment:
Before it failed loud and clear when this didn’t work, but now `min_stack_size` returns `PTHREAD_STACK_MIN` instead which might or might not be a correct value to use in the stack size calculations.

/me shrugs.